snippets/json-schema.code-snippets

```diff
@@ -1,118 +1,118 @@
 {
-	"Number property": {
+	"Snowboard: Number property": {
         "prefix": ["number"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"number\"", "}"],
-        "description": "A for loop."
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"number\",", "\t\"description\": \"${2:Description goes here}\"", "}${0}"],
+        "description": "A property of type 'number'."
     },
-    "String property": {
+    "Snowboard: String property": {
         "prefix": ["string"],
```

Comment:
The whole prefix is not needed for it to trigger. The dropdown shows up as soon as it is able to reasonably differentiate. 
The problem with adding multiple prefixes which looks similar (like ["integer", "int"]) is that it will show up multiple entries for each of the matching prefix in the dropdown. So in this case when I type `int` it will show up two entries like so 
```
int      Snowboard: integer property
integer  Snowboard: integer property
```

Note: The autocomplete search is quite good. If you type `dt`, it will pickup date-time as well.